src/vunnel/providers/nvd/__init__.py

```diff
@@ -38,6 +38,7 @@ def __str__(self) -> str:
 
 
 class Provider(provider.Provider):
+    # TODO: why not int(schema.NVDSchema().major_version) (@weston)
```

Comment:
Before we can merge this we should reconcile what version is used (both here and in the listing file)

Reply:
this has been changed to conceptually split up `version` of a provider (which includes processing behavior + results data shape) and the `distribution_version` of a provider (which strictly includes results data shape).